src/js/components/FileInput/stories/Custom.js

```diff
@@ -18,6 +18,12 @@ const customTheme = {
         horizontal: '8px',
       },
     },
+    message: {
+      maxFileMessage: {
```

Comment:
Can we just call this `maxFile` since Message is slightly redundant?

Reply:
Sure